src/main/java/com/github/avasin/yarmij/messages/handlers/RmiInvokeMethodMessageHandler.java

```diff
@@ -75,7 +75,7 @@ public void accept(@Nonnull RmiConnection transport,
                             ((InvocationTargetException)ex).getTargetException() :
                             ex;
             logger.error("Cannot execute {}#{} with {} arguments",
-                            implementationClass.getSimpleName(), methodName, message.getArgs(),
+                            implementationClass.getSimpleName(), signature.getMethodName(), message.getArgs(),
```

Comment:
Please, add/update toString method in RmiSignature and pass everywhere signature itself.